package.json

```diff
@@ -32,7 +32,7 @@
     "webworkify": "^1.2.1",
     "yo-yo": "^1.2.2",
     "yo-yoify": "^3.3.0",
-    "ethereum-remix": "0.0.2-alpha.0.0.4"
+    "ethereum-remix": "0.0.2-alpha.0.0.7"
```

Comment:
You could also just increase the patch level, `0.0.2` to `0.0.3`.

Also if you dont really need an npm release (yet) and the only reason it is on npm to use it here, you could replace the whole with a github url.  Npm will download it that way also.